lib/ecl/ecl_file_kw.c

```diff
@@ -323,4 +336,88 @@ void ecl_file_kw_inplace_fwrite( ecl_file_kw_type * file_kw , fortio_type * fort
 }
 
 
+void ecl_file_kw_fwrite( const ecl_file_kw_type * file_kw , FILE * stream ) {
+  int header_length = strlen( file_kw->header );
+  for (int i=0; i < ECL_STRING8_LENGTH; i++) {
+    if (i < header_length)
+      fputc( file_kw->header[i], stream );
+    else
+      fputc( ' ' , stream );
+  }
```

Comment:
Just to help me understand... by reading the piece of code above, I got the impression that the header_length is expected to be less than  ECL_STRING8_LENGTH (which seems to be 8). I was wondering, does the header have usually a specific length?